system_tests/vision.py

```diff
@@ -392,3 +393,64 @@ def test_detect_safe_search_filename(self):
         self.assertEqual(len(safe_searches), 1)
         safe_search = safe_searches[0]
         self._assert_safe_search(safe_search)
+
+
+class TestVisionClientText(unittest.TestCase):
+    DESCRIPTIONS = (
+        'Do',
```

Comment:
Why are the component words here? AFAICT, you only need to test against the full phrase.